src/pkg/systemartifact/cleanupcriteria.go

```diff
@@ -0,0 +1,45 @@
+package systemartifact
+
+import (
+	"context"
+	"github.com/goharbor/harbor/src/jobservice/logger"
+	"github.com/goharbor/harbor/src/lib/q"
+	"github.com/goharbor/harbor/src/pkg/systemartifact/dao"
+	"github.com/goharbor/harbor/src/pkg/systemartifact/model"
+	"time"
+)
+
+var (
+	DefaultCleanupWindowSeconds = 86400
+)
+
+// CleanupCriteria provides an interface that can be implemented
+// by consumers of the system artifact management framework to
+// provide a custom clean-up criteria. This allows producers of the
+// system artifact data to control the lifespan of the generated artifact
+// records and data.
+// Every system data artifact produces must register a cleanup criteria.
+
+type CleanupCriteria interface {
+	List(ctx context.Context) ([]*model.SystemArtifact, error)
+}
+
+var DefaultCleanupCriteria = NewCleanupCriteria()
```

Comment:
Default()? then the caller will be cleanupcriteria.Default().

This rule could apply to other method name in this go file.

Reply:
Please add some comments about default beharior

Reply:
Done. I have renamed the method to `Default()`. I however could not get the second part of the comment `This rule could apply to other method name in this go file.`.